R/zero-curve-class.R

```diff
@@ -190,7 +190,8 @@ Interpolation <- function(method, what) {
     constant_zeros = "Constant",
     constant_forwards = "LogDF",
     linear_zeros = "Linear",
-    natural_cubic_zeros = "Cubic"
+    natural_cubic_zeros = "Cubic",
+    linear_time_variance = "Linear"
```

Comment:
Do you really need a new interpolation method called `LinearTimeVarInterpolation`? Isn't this simply LinearInterpolation applied to time weighted variance?

Reply:
Even though the name says Linear, but the actual interpolation is Linear on time, Cubic spline on strike. Including the whole description will blow up the length of the method name.  The actual interpolation is two-fold: 1st interpolation on the time, and a 2nd interpolation on the strikes.